go/vt/sidecardb/schema/schemaengine/tables.sql

```diff
@@ -16,8 +16,8 @@ limitations under the License.
 
 CREATE TABLE IF NOT EXISTS tables
 (
-    TABLE_SCHEMA varchar(64) NOT NULL,
-    TABLE_NAME varchar(64) NOT NULL,
+    TABLE_SCHEMA varbinary(64) NOT NULL,
+    TABLE_NAME varbinary(64) NOT NULL,
```

Comment:
I think this is not correct? `varbinary(64)` means a maximum byte size of 64 bytes, not 64 characters which would break table names with multibyte characters.

I think the correct way here is to not change the type, but just the collation here to be case sensitive instead? Just for these specific columns?

Reply:
Sounds like a good idea, I'll make the change

Reply:
Done.